### PR TITLE
#453 subscribe to all issues

### PR DESCRIPTION
**Pull request by KZavorotna**
Related to: 
https://github.com/withthegrid/platform/pull/743
https://github.com/withthegrid/platform-sdk-private/pull/9
https://github.com/withthegrid/platform-client/pull/593

### CODE REVIEW

File: src/routes/environment/find.ts [hunk 1]
```diff
@@ -34,6 +35,9 @@ const controllerGeneratorOptions: ControllerGeneratorOptionsWithoutClientOrSuppl
       environment: environmentSchema.required(),
       environmentRights: Joi.array().items(Joi.string()).required().example(['STATIC', 'USERS'])
         .description('See the getting started section about rights'),
+      notificationLevel: Joi.number().allow(null).required()
```
Comment (everhardt) [inline src/routes/environment/find.ts:38]: This should be an exception to my remark that "everywhere where `environmentRights` are supplied in a response, also supply `notificationLevel: 0 | 1 | 2 | null`". The `environmentRights` here are not about the user performing the request, but the user of the row and I don't think it makes sense that the requesting user knows about that. So can you drop `notificationLevel` again here?

File: src/routes/authentication/machine-login.ts [hunk 2]
```diff
@@ -15,6 +15,7 @@ interface Response {
   user: User;
   environment?: Environment;
   environmentRights?: string[];
+  notificationLevel: 0 | 1 | 2 | null;
```
Comment (everhardt) [inline src/routes/authentication/machine-login.ts:18]: Can it be made optional without any dire consequences, and the Joi schema as well?

File: src/routes/settings/get.ts [hunk 3]
```diff
@@ -11,6 +11,7 @@ interface Response {
   environment?: Environment;
   environmentRights?: string[];
   user: User;
+  notificationLevel: 0 | 1 | 2 | null;
```
Comment (everhardt) [inline src/routes/settings/get.ts:14]: Can it be made optional without any dire consequences, and the Joi schema as well?

**Comment by KZavorotna** [comment]
Ready for review

**Comment by everhardt** [comment]
Thanks! Merged this into https://github.com/withthegrid/platform-sdk/commit/d4d65f2c61e70f415b45309cdf7a71513d47b03e, so will close this PR

**Pull request closed without merging**